content/en/docs/concepts/workloads/autoscaling.md

```diff
@@ -0,0 +1,55 @@
+---
+reviewers: []
+title: Autoscaling Workloads
+description: >-
+  With Autoscaling, you can automatically update your workloads or infrastructure in one way or another. This allows your cluster to react to changes in resource demand more elastically and efficiently.
+content_type: concept
+weight: 40
+hide_summary: true # Listed separately in section index
+---
+
+<!-- overview -->
+
+The concept of _Autoscaling_ in Kubernetes refers to the ability to automatically update the available
+resources of your cluster. This can be either a replication controller (for example a
+{{< glossary_tooltip text="Deployment" term_id="deployment" >}} or
+{{< glossary_tooltip text="ReplicaSet" term_id="replica-set" >}}), or the cluster infrastructure
+itself (for example the number of {{< glossary_tooltip text="Nodes" term_id="node" >}}).
```

Comment:
Avoid using the term “replication controller”; it's likely to mislead. We'd now say something like “an object that lets you manage a set of Pods”.